src/layouts/Login/LoginPage.tsx

```diff
@@ -44,194 +44,191 @@ const LOGIN_GRID_LAYOUT: Pick<
-                _hover={{
-                  color: "text.link.dark",
-                  textDecorationLine: "underline",
-                }}
-              >
-                Contact Us
-              </Link>
-              <Link
-                href="https://go.gov.sg/isomercms-guide/"
-                rel="noopener noreferrer"
-                target="_blank"
-                textDecorationLine="none"
-                color="text.link.dark"
-                _hover={{
-                  color: "text.link.dark",
-                  textDecorationLine: "underline",
-                }}
-              >
-                Guide
-              </Link>
-              <Link
-                href="https://www.tech.gov.sg/report_vulnerability/"
-                rel="noopener noreferrer"
-                target="_blank"
-                textDecorationLine="none"
-                color="text.link.dark"
-                _hover={{
-                  color: "text.link.dark",
-                  textDecorationLine: "underline",
-                }}
-              >
-                Report vulnerability
-              </Link>
-            </HStack>
-          </Flex>
-        </GridItem>
-      </Grid>
-    </Flex>
+    <VStack gap="2.5rem" alignItems="start" width="65%">
+      <Text fontSize="2.5rem" color="text.title.brand" textStyle="display-2">
+        Rapidly build & launch informational sites
+      </Text>
+      <InlineMessage>
+        We’re moving in phases from GitHub IDs to email addresses as the login
+        method. If you can’t log in with your GitHub ID, please use your email
+        address.
+      </InlineMessage>
+      <Tabs width="100%">
+        <TabList>
+          <Tab>Github Login</Tab>
+          <Tab>Email Login</Tab>
+        </TabList>
+        <TabPanels pt="2rem">
+          <TabPanel>
+            <Button
+              as={Link}
+              rel="noopener noreferrer"
+              textDecoration="none"
+              _hover={{
+                textDecoration: "none",
+                bgColor: "primary.600",
+              }}
+              href={`${process.env.REACT_APP_BACKEND_URL_V2}/auth/github-redirect`}
+            >
+              <Text color="white">Log in with GitHub</Text>
+            </Button>
+          </TabPanel>
+          <TabPanel>
+            {email ? (
+              <OtpForm
+                email={email}
+                onSubmit={handleVerifyOtp}
+                onResendOtp={handleResendOtp}
+              />
+            ) : (
+              <LoginForm onSubmit={handleSendOtp} />
+            )}
+          </TabPanel>
+        </TabPanels>
+      </Tabs>
+      <Text color="text.helper" fontSize="0.625rem">
+        By clicking “Log in”, you are acknowledging and agreeing to Isomer’s{" "}
+        <Link
+          href="https://guide.isomer.gov.sg/terms-and-privacy/terms-of-use"
+          rel="noopener noreferrer"
+          target="_blank"
+          fontSize="0.625rem"
```

Comment:
Nit: `fontSize` is not needed - the `Text` parent already has a `fontSize` of `0.625rem`

Reply:
You're right - fixed in https://github.com/isomerpages/isomercms-frontend/pull/996/commits/9480054f6a331012456d57ca3448512d15398f42